answers/package-search-naming.md

```diff
@@ -0,0 +1,23 @@
+# Problem
+## Context (What I'm Trying To Accomplish)
+
+I want my package, elm-random-thing, to show up when someone searches for JSON.
```

Comment:
It would be helpful to give a more concrete example. What does this package have to do with JSON? I know you meant "I want my package to show up when it's relevant" but it comes off as "I want to name grab all the commonly searched terms".